app/config/config_dev.yml

```diff
@@ -5,6 +5,12 @@ framework:
     router:   { resource: "%kernel.root_dir%/config/routing_dev.yml" }
     profiler: { only_exceptions: false }
 
+doctrine:
+    dbal:
+        driver: pdo_sqlite
+        path: %kernel.root_dir%/Resources/sqlite.db
```

Comment:
This path is not a writable location, which is very bad for people running Packagist through php-fpm locally rather than using the builtin webserver, as it will mess up permissions.
and btw, it is not an ignored location either, which means people are likely to commit their DB, which is a bad idea